templates/cloudformation/aws_agent_with_telemetry_collector.yaml

```diff
@@ -0,0 +1,136 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
this parent stack is an example that deploys the MC Agent with the Telemetry Collector using an existing VPC in AWS

Reply:
slack [context](https://montecarlodata.slack.com/archives/C0975GJ4VNC/p1753982243341599)